docs/configuring-a-resource.md

```diff
@@ -888,7 +890,7 @@ initializers for a resource.
 [handle dependencies]: https://docs.crossplane.io/master/concepts/managed-resources/#referencing-other-resources
 [user]: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_access_key#user
 [generate reference resolution methods]: https://github.com/crossplane/crossplane-tools/pull/35
-[configuration]: https://github.com/crossplane/upjet/blob/main/pkg/config/resource.go#L123
+[configuration]: https://github.com/crossplane/upjet/blob/942508c5370a697b1cb81d074933ba75d8f1fb4f/pkg/config/resource.go#L172
```

Comment:
Thanks for linking to specific commit trees, instead of the main branch 🙏 Doing so prevents links going out of context, when the linked resources are updated.